service_identity/src/main/kotlin/org/thechance/service_identity/api/model/request/CreateUserRequest.kt

```diff
@@ -0,0 +1,10 @@
+package org.thechance.service_identity.api.model.request
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class CreateUserRequest(
```

Comment:
I think `CreateUserRequest` is giving more as a function name more than a class name.